grid-opdracht.html

```diff
@@ -6,6 +6,12 @@
     <title>Les 3 - Opdracht CSS Grid</title>
 </head>
 <body>
-
+<div class="container">
+    <div class="item" id="vak-a">A</div>
+    <div class="item" id="vak-b">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam sagittis, velit vitae posuere dapibus, mi leo faucibus est, ut mollis quam turpis id est. Vivamus ultricies scelerisque scelerisque. Pellentesque risus eros, pellentesque id nisl ut, eleifend fermentum purus. Vestibulum quis dolor justo. Nunc aliquet ex id ante luctus pharetra. Mauris iaculis placerat erat non tincidunt. Etiam est arcu, tincidunt vel placerat nec, varius at orci. Aenean semper elit vel ipsum placerat varius. Vestibulum ut nunc eu augue maximus gravida eget ac lacus. Mauris sollicitudin justo lectus. Morbi ullamcorper tempus mauris, et mattis leo varius sit amet. Praesent nec accumsan ante. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum aliquet convallis egestas.</div>
+    <div class="item" id="vak-c">C</div>
+    <div class="item" id="vak-d">D</div>
+    <div class="item" id="vak-e">E</div>
+    <div class="item" id="vak-f">F</div>
 </body>
 </html>
```

Comment:
Alles keurig volgens opdracht!
```suggestion
</html>
```